packages/web3-shared/evm/src/types/index.ts

```diff
@@ -385,6 +385,7 @@ export interface Transaction {
     data?: string
     nonce?: number
     chainId?: number
+    type?: string
```

Comment:
Learn more: https://docs.infura.io/api/networks/ethereum/concepts/transaction-types

```suggestion
    type?: '0x0' | '0x1' | '0x2'
```